src/components/ui/SearchSort.tsx

```
@@ -0,0 +1,103 @@
+  return (
+    <div className={`relative ${className}`}>
+      <button
+        onClick={() => setIsOpen(!isOpen)}
+        className={`inline-flex items-center gap-2 whitespace-nowrap rounded-lg px-4 py-2.5 text-caption transition-colors ${
+          isSorted
+            ? "border-2 border-primary bg-primary-10 text-muted-foreground focus:outline-none"
+            : isOpen
+              ? "border-2 border-primary bg-white text-muted-foreground focus:outline-none"
+              : "border border-[hsl(var(--border))] bg-white text-muted-foreground hover:border-neutral-20 focus:border-transparent focus:outline-none focus:ring-2 focus:ring-primary"
+        }`}
+      >
```

Biome (2.3.13)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)